actionmailbox/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Fixed ingress controllers' ability to accept emails that contain no UTF-8 encoded parts.
+
+    Fixes #46297.
+
+    *Jan Honza Sterba*
```

Comment:
Not trying to de-credit you but I've opened #47097 since I've been told we don't put bug fixes in the ChangeLog.

Also mentioned [here](https://edgeguides.rubyonrails.org/contributing_to_ruby_on_rails.html#updating-the-changelog):

> You should add an entry to the top of the CHANGELOG of the framework you modified if you're adding or removing a feature, or adding deprecation notices. Refactorings, minor bug fixes, and documentation changes generally should not go to the CHANGELOG.